tests/v2/validation/steve/vai/vai_test.go

```diff
@@ -59,7 +60,7 @@ func (v *VaiTestSuite) TearDownSuite() {
 func (v *VaiTestSuite) enableVai() error {
 	logrus.Info("Enabling VAI caching")
 	startTime := time.Now()
-	err := enableVAI(v.client, &v.vaiEnabled, &v.once)
+	err := vaiaction.EnableVAI(v.client, &v.vaiEnabled, &v.once)
```

Comment:
nit; can the name of this just be vai.EnableVAI and update the shepherd one to shepherdvai? It it just a little hard to read this way since there's a few vowels all together.

Reply:
also, where is v.once set? I am not up to doate on how sync.Once works but right now I don't see where v.once is set before this line..

Reply:
its defined in the test struct